python/lsst/ts/imsim/closed_loop_task.py

```diff
@@ -1037,8 +1062,12 @@ def run_img(
             Raw seeing in arcsec.
         imsim_log_file : str
             Location to save imsim log output.
+        wep_estimator_method : str
+            Specify the method used to calculate Zernikes in ts_wep.
+            Options are "Tie" or "danish".
```

Comment:
Is it intentional that "Tie" starts from uppercase, and "danish" does not?  Would it hurt to use "tie" and "danish"?

Reply:
ok, reading below I think it's just a typo, as `add_argument` has `tie` and `danish` as I suspected

Reply:
Yeah, typo. Thanks for the catch. Fixed now.